js-pkg/create-y-sweet-app/src/templates/remix/.env.local

```diff
@@ -0,0 +1 @@
+Y_SWEET_CONNECTION_STRING="ys://127.0.0.1:8080"
```

Comment:
Just noticing that there's a `.env.default` in the nextjs example but not one here. Is that intentional?